docs/user_guide/wflow_model_setup.rst

```diff
@@ -26,84 +26,124 @@ setup methods are exposed. Click on
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_config_output_timeseries`
+      - This method add new variable/column to the netcdf/csv output section of the toml based on a selected gauge/area map.
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_precip_forcing`
       -  Setup gridded precipitation forcing at model resolution.
-    * - :py:func:`~WflowSbmModel.setup_precip_from_point_timeseries`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_precip_from_point_timeseries`
       -  Setup precipitation forcing from station data at model resolution.
-    * - :py:func:`~WflowSbmModel.setup_temp_pet_forcing`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_temp_pet_forcing`
       -  Setup gridded temperature and optionally compute reference evapotranspiration forcing at model resolution.
-    * - :py:func:`~WflowSbmModel.setup_pet_forcing`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_pet_forcing`
       -  Setup gridded reference evapotranspiration forcing at model resolution.
-    * - :py:func:`~WflowSbmModel.setup_constant_pars`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_constant_pars`
       -  Setup constant parameter maps for all active model cells.
-    * - :py:func:`~WflowSbmModel.setup_allocation_areas`
+      - None
+    * - :py:meth:`~WflowSbmModel.setup_allocation_areas`
       -  Create water demand allocation areas.
-    * - :py:func:`~WflowSbmModel.setup_allocation_surfacewaterfrac`
+      - :py:meth:`~WflowSbmModel.setup_rivers`
+    * - :py:meth:`~WflowSbmModel.setup_allocation_surfacewaterfrac`
       -  Create fraction of surface water used for allocation of the water demands.
-    * - :py:func:`~WflowSbmModel.setup_domestic_demand`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_domestic_demand`
       -  Create domestic water demand from grid.
-    * - :py:func:`~WflowSbmModel.setup_domestic_demand_from_population`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_domestic_demand_from_population`
       -  Create domestic water demand using demand per capita and gridded population.
-    * - :py:func:`~WflowSbmModel.setup_other_demand`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_other_demand`
       -  Create other water demand (eg industry, livestock).
-    * - :py:func:`~WflowSbmModel.setup_irrigation`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_irrigation`
       -  Create irrigation areas and trigger for paddy and nonpaddy crops from a raster file.
-    * - :py:func:`~WflowSbmModel.setup_irrigation_from_vector`
+      - :py:meth:`~WflowSbmModel.setup_lulcmaps`, :py:meth:`~WflowSbmModel.setup_laimaps`
```

Comment:
We have 3 for lulc, also ``setup_lulcmaps_from_vector`` and ``setup_lulcmaps_with_paddy``
```suggestion
      - :py:meth:`~WflowSbmModel.setup_lulcmaps` or equivalent, :py:meth:`~WflowSbmModel.setup_laimaps` or equivalent
```